PS8lite.Rmd

```diff
@@ -187,23 +188,58 @@ MiscVal + MoSold + YrSold" %>%
 Fit one Random Forest model `model_rf` using any approach you want.
 
 ```{r}
+# Define cross-validation settings: 10-fold CV
+fit_control <- trainControl(method = "cv", number = 10)
+# Setup search grid of "tuning" parameters
+mtry <- 2:5
+tunegrid <- expand.grid(.mtry = mtry)
```

Comment:
Not sure if you tried to up the max limit of `mtry`. But if you set it to 36 (the number of predictors) this is a great example of a problem where computational constraints become an issue! In other words, method A might yield "better predictions" than method B, but if method A takes a week to run and method B takes only a few minutes, you might sacrifice a little predictive power for speed!